discover/deployment/articles/01-installing-search-engine/01-configuring-elasticsearch/03-logging.markdown

```diff
@@ -0,0 +1,124 @@
+# Elasticsearch Logging
+
+When you first start @product@, an embedded Elasticsearch server starts so that
+search works out of the box. The embedded search engine is not suitable for
+production, but is useful for testing and development.
+
+The @product@ log was slimmed down by removing INFO level Elasticsearch logs for
+the embedded Elasticsearch. WARN and ERROR logs are still displayed. If you
+miss hearing from Elasticsearch during portal startup, this article shows you
+how to enable the INFO log level for the embedded Elasticsearch server.
+
++$$$
+
+**Note:** These instructions show you how to adjust the embedded Elasticsearch
+server's logs. Logging for @product@'s search functionality is configurable via
+the 
+[Log Levels](/discover/portal/-/knowledge_base/7-1/server-administration) 
+screen at Control Panel &rarr; Configuration &rarr; Server Administration &rarr;
+Log Levels tool. Narrow down the list to include only the search classes and
+packages by searching for _com.liferay.portal.search_.
+
+To adjust logging for a remote Elasticsearch server, see 
+[Elastic's documentation](https://www.elastic.co/guide/en/elasticsearch/reference/6.1/logging.html).
+
+$$$
+
+Here's an example log message that _is_ displayed by default:
```

Comment:
might be helpful to make it a little more clear what type of logs (portal vs elastic) you are giving an example of. ie

Here's an example **Portal** log message that _is_ displayed by default:

Here's an example **Elasticsearch** log message that _isn't_ displayed by default: